examples/complete/main.tf

```diff
@@ -1,3 +1,4 @@
+# trunk-ignore-all(trivy/AVD-AWS-0178): We don't need have VPC Flow logs.
```

Comment:
What I was going to update here (and successfully forgot to complete 😑) is to redo the example without referencing to context.tf (maybe worth leaving it commented for the users who utilize this pattern). Do you want me to add this to your branch? Or you'll update it by yourself?

Reply:
@gberenice Hm -- I'm glad you're thinking of this as I always forget to update the example. I think what I'll do is update the readme for the two use-cases: 1) passing `primary_tag` and 2) passing in `context.tf` label elements to create the module.this.id